src/nwb_conversion_tools/utils/json_schema.py

```diff
@@ -12,7 +12,7 @@
 
 def get_base_schema(tag=None, root=False, id_=None, **kwargs) -> dict:
     """Return the base schema used for all other schemas."""
-    base_schema = dict(required=[], properties={}, type="object", additionalProperties=False)
+    base_schema = dict(required=[], properties={}, type="object", additionalProperties=True)
```

Comment:
This one line can actually have  variety of unintended consequences since it affects the base of every schema.

Is this actually needed, or can you just set `additionalProperties=True` in the one part of the sub-schema where it is actually needed?